mypy_django_plugin/lookups.py

```diff
@@ -0,0 +1,151 @@
+import dataclasses
+from typing import Union, List
+
+from mypy.nodes import TypeInfo
+from mypy.plugin import CheckerPluginInterface
+from mypy.types import Type, Instance
+
+from mypy_django_plugin import helpers
+from mypy_django_plugin.transformers.fields import get_private_descriptor_type
```

Comment:
Maybe move `get_private_descriptor_type` to `helpers` as it's used in more than one file now?